src/omnicore/fees.cpp

```diff
@@ -43,6 +43,11 @@ void COmniFeeCache::UpdateDistributionThresholds()
         }
     }
 
+    // override above and use a fixed total count of 600,000 for OMNI & TOMNI
+    // this is done to remove the need for recalculating distribution thresholds every block (where dev OMNI fractionally changes the total number of tokens)
+    int64_t primaryTokenOverride = 600000 / OMNI_FEE_THRESHOLD;
```

Comment:
Basically `600000` is a ballpark count of the total number of Omni tokens (give or take).  This is the hack I used to time what the impact of doing the recalculations every block was.

I was supposed to commit the bit where UpdateDistributionThresholds() is property aware and only calculates it for the token changed, sorry dude let me fix this up.

I was copying code from a working directory, obviously didn't do it right :(